.ci/jenkins_job_build.sh

```diff
@@ -259,11 +259,43 @@ case "${CI_JOB}" in
 	export KUBERNETES="yes"
 	export experimental_kernel="true"
 	;;
-
+"VFIO")
+	init_ci_flags
+	export CRIO="no"
+	export CRI_CONTAINERD="yes"
+	export CRI_RUNTIME="containerd"
+	export KATA_HYPERVISOR="qemu"
+	export KUBERNETES="yes"
+	export OPENSHIFT="no"
+	;;
 esac
 "${ci_dir_name}/setup.sh"
 
-if [ "${METRICS_CI}" == "false" ]; then
+if [ "${CI_JOB}" == "VFIO" ]; then
+	pushd "${GOPATH}/src/${tests_repo}"
+	ci_dir_name=".ci"
+
+	echo "Installing initrd image"
+	export AGENT_INIT=yes TEST_INITRD=yes OSBUILDER_DISTRO=alpine
+	sudo -E PATH=$PATH "${ci_dir_name}/install_kata_image.sh"
+
+	# Do not install cloud-hypervisor, issue:
+	# https://github.com/kata-containers/kata-containers/issues/862
```

Comment:
https://github.com/kata-containers/kata-containers/issues/862 is now closed, so can we uncomment?

Reply:
:astonished: you are right